### PR TITLE
ktlint 0.47.1

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Porkepix** [comment]
Should be compatible with latest `openjdk`, their repo tests against latest JDKs.
Not sure of the changes I did on `write_env_script` though.

**Comment by chenrui333** [comment]
- relates to https://github.com/pinterest/ktlint/pull/1617

File: Formula/ktlint.rb [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
+  depends_on "openjdk"
 
   def install
     libexec.install "ktlint"
     (libexec/"ktlint").chmod 0755
-    (bin/"ktlint").write_env_script libexec/"ktlint",
-                                    Language::Java.java_home_env("11").merge(
-                                      PATH: "#{Formula["openjdk@11"].opt_bin}:${PATH}",
-                                    )
+    (bin/"ktlint").write_env_script libexec/"ktlint", Language::Java.java_home_env
```
Comment (carlocab) [inline Formula/ktlint.rb:17]: Might still need `PATH`, since `openjdk` is also keg-only.

**Review by carlocab — APPROVED** [review]

File: Formula/ktlint.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/ktlint.rb:17]: I see.

> Test seems happy though.

probably because `shadowed_by_macos`?

Comment (chenrui333) [reply Formula/ktlint.rb:17]: never mind, after a bit reading, I dont think PATH change is needed.

**Pull request closed without merging**